packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -188,6 +189,59 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	}
 }
 
+async function downloadWordPressAssets(php: PHP) {
+	if (!php.requestHandler) {
+		logger.warn('No PHP request handler available');
+		return;
+	}
+	const wpVersion = await getLoadedWordPressVersion(php.requestHandler);
+	const staticAssetsDirectory = wpVersionToStaticAssetsDirectory(wpVersion);
+	if (!staticAssetsDirectory) {
+		logger.warn('No static assets directory for WordPress', wpVersion);
```

Comment:
When would this happen in practice? Would it mean we're not running a minified build? Or that something is wrong with the build we are running? Let's communicate that explicitly in the error message, whether it's thrown or logged.

Reply:
This happens when we try to load an unsupported version of WordPress. In practice, this could happen if you are mounting a local directory with an outdated version of WP.

I removed this log message in [1aad0d0](https://github.com/WordPress/wordpress-playground/pull/1532/commits/1aad0d0fe9822f83f292ef6a1a6dc822b20c42fb) because it doesn't provide any value to the user.